src/services/ratingService.js

```diff
@@ -0,0 +1,27 @@
+
+/* eslint-disable no-plusplus */
+
+const filterRatingArray = (array)=>{
+    const newArray = []
+    for(let i= 0; i< array.length; i++){
+      newArray.push(array[i].service_rating)
+    }
+    return newArray;
+  }
+const classifyRating =(ratings)=>{
+    const serviceRating = filterRatingArray(ratings)
+    const count = {};
+for (let i=0; i < serviceRating.length; i++) {
+  if (count[serviceRating[i]]) {
+    count[serviceRating[i]] += 1;
+  } else {
+    count[serviceRating[i]] = 1;
+  }
+}
+return count;
+}
+const round = (num) =>{
+    const m = Number((Math.abs(num) * 10).toPrecision(15));
+    return Math.round(m) / 10 * Math.sign(num);
+}
+export {classifyRating, round}
```

Comment:
**1**. 2 users have rated the same accommodation (accom1)
**2**.  user1 has rated accom1 to 3
**3**.  user2 has rated accom1 to 5
**4**. the database holds these data and I don't see how you are interacting with them in your service file to find the rating (if it is the average)

Reply:
You pass the list of ratings to the average function from the service file and you get back the average of ratings